python/mxnet/_numpy_op_doc.py

```diff
@@ -22,37 +22,73 @@
 
 def _np_reshape(a, newshape, order='C'):
     """
-    reshape(a, newshape, order='C')
-
     Gives a new shape to an array without changing its data.
 
     Parameters
     ----------
     a : ndarray
         Array to be reshaped.
-    newshape : int or tuple of ints
+    newshape : int, tuple of ints or list of ints
         The new shape should be compatible with the original shape. If
         an integer, then the result will be a 1-D array of that length.
         One shape dimension can be -1. In this case, the value is
         inferred from the length of the array and remaining dimensions.
-    order : {'C'}, optional
+    order : 'C', optional
         Read the elements of `a` using this index order, and place the
         elements into the reshaped array using this index order.  'C'
         means to read / write the elements using C-like index order,
         with the last axis index changing fastest, back to the first
-        axis index changing slowest. Other order types such as 'F'/'A'
-        may be added in the future.
+        axis index changing slowest. We will add 'F'/'A' order types
```

Comment:
Would reword to not make promises. Link to the issue/proposal doc.

Reply:
done